modules/client-configuration/pages/cobbler.adoc

```diff
@@ -82,6 +51,8 @@ You must have administrative access to the network's DHCP server, or be able to
 
 If you have an existing DHCP server, you will need to edit the DHCP configuration file so that it points to the Cobbler server and PXE boot image.
 
+
+
 .Procedure: Configuring DHCP for Cobbler
```

Comment:
Can we declare this as "Example in case your use the ISC DHCP Server"

Reply:
ok, I changed it.